### ISSUE TITLE
[Tooltip] Build error with TypeScript and Webpack

### ISSUE THREAD

**Issue by teddywing** [material-components/material-components-web#6657]
## Bug report
I'm getting a build error when using the `@material/tooltip` 8.0.0 package (https://www.npmjs.com/package/@material/tooltip/v/8.0.0) along with TypeScript and Webpack.

This appears to be caused by the presence of `.ts` TypeScript files in the NPM package.

### Steps to reproduce
1. Clone this Gist: https://gist.github.com/teddywing/245011c223283d3203038a3738625619

		$ git clone git@gist.github.com:245011c223283d3203038a3738625619
		$ cd 245011c223283d3203038a3738625619

2. Install packages

		$ npm install

3. Build with Webpack

		$ ./node_modules/.bin/webpack

### Actual behavior
```
$ ./node_modules/.bin/webpack
[webpack-cli] Compilation finished
assets by status 1.95 KiB [cached] 1 asset
runtime modules 657 bytes 3 modules
cacheable modules 25.1 KiB
  modules with errors 23.8 KiB [errors]
    ./node_modules/@material/tooltip/component.ts 5.55 KiB [built] [code generated] [1 error]
    ./node_modules/@material/tooltip/foundation.ts 15.9 KiB [built] [code generated] [1 error]
    ./node_modules/@material/tooltip/constants.ts 2.32 KiB [built] [code generated] [1 error]
  ./index.ts 104 bytes [built] [code generated]
  ./node_modules/@material/tooltip/index.js 1.22 KiB [built] [code generated]

ERROR in ./node_modules/@material/tooltip/component.ts 31:44
Module parse failed: Unexpected token (31:44)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| import {MDCTooltipFoundation} from './foundation';
|
> export class MDCTooltip extends MDCComponent<MDCTooltipFoundation> {
|   static attachTo(root: Element) {
|     return new MDCTooltip(root);
 @ ./node_modules/@material/tooltip/index.js 23:0-28 23:0-28
 @ ./index.ts 1:0-47 2:4-14

ERROR in ./node_modules/@material/tooltip/constants.ts 24:0
Module parse failed: The keyword 'enum' is reserved (24:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
|
> enum CssClasses {
|   SHOWN = 'mdc-tooltip--shown',
|   SHOWING = 'mdc-tooltip--showing',
 @ ./node_modules/@material/tooltip/index.js 25:0-28 25:0-28
 @ ./index.ts 1:0-47 2:4-14

ERROR in ./node_modules/@material/tooltip/foundation.ts 40:55
Module parse failed: Unexpected token (40:55)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| } = CssClasses;
|
> export class MDCTooltipFoundation extends MDCFoundation<MDCTooltipAdapter> {
|   static get defaultAdapter(): MDCTooltipAdapter {
|     return {
 @ ./node_modules/@material/tooltip/index.js 24:0-29 24:0-29
 @ ./index.ts 1:0-47 2:4-14

webpack 5.9.0 compiled with 3 errors in 3248 ms
```

### Expected behavior
The build succeeds.

### Your Environment:

| Software         | Version(s) |
| ---------------- | ---------- |
| MDC Web          | 8.0.0 |
| Browser          | Google Chrome 87.0.4280.63 |
| Operating System | macOS 10.15 |

### Possible solution
I've noticed other `@material` NPM packages don't include `.ts` files, and wonder if they should be removed from the `@material/tooltip` package.

**Comment by joyzhong** [comment]
This has been fixed (via #6634 ).

**Comment by teddywing** [comment]
Thanks!
